### ISSUE TITLE
pdf file hides other page tabs' content

### ISSUE THREAD

**Issue by vbr** [OtterBrowser/otter-browser#991]
Hi,
It seems, that some non-html files open directly in Otter somehow override the other page tabs and effectively hide all other tab content - e.g. a pdf files shown in this way remains in view, even if other page tabs are activated (the address bar info changes accordingly). The normal behaviour and access to other tabs is restored only after closing the "dominant" pdf page. (I noticed this with different pdf viewers (Acrobat viewer, pdf Architect)
(Win7, Otter webkit 32bit, weekly 96).
Regards,
    vbr

**Comment by Emdek** [comment]
@vbr, it appears that old issue in QtWebkit had reappeared or wasn't completely fixed in Qt 5.4.2.
I've added comment to upstream report:
https://bugreports.qt.io/browse/QTBUG-43024

Duplicate of #150.
